support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -212,10 +217,12 @@ export function ContributionsOrderSummary({
 			<div css={[summaryRow, rowSpacing, boldText, totalRow(!!tsAndCs)]}>
 				<p>Total</p>
 				<p>
-					{totalWithFrequency(
-						simpleFormatAmount(currency, total),
-						contributionType,
-					)}
+					{productDescription && `${total}/${productDescription.frequency}`}
+					{!productDescription &&
+						totalWithFrequency(
+							simpleFormatAmount(currency, total),
+							contributionType,
+						)}
```

Comment:
This is the only change to a live file.

I have done it like this to decouple it from concepts like `contributionType`[^1] giving more control to the instantiating component. If we like this a secondary step would be to remove those `props` completely.

[^1]: `'ONE_OFF' | 'MONTHLY' | 'ANNUAL'`